Builder Pattern/src/Builder.java

```diff
@@ -0,0 +1,61 @@
+public class Builder {
+     String firstName;
+     String lastName;
+     int age;
+     String address;
+     String city;
+     String state;
+     String country;
+     String phone;
+     String email;
+     String occupation;
+
+    public Builder(String firstName, String lastName) {
```

Comment:
why these 2 attributes are made mandatory. is it required?

Reply:
No it was not mentioned.
But I wanted to show that we will be having both mandatory and optional fields.